libs/components/datetime/src/lib/modules/date-range-picker/date-range-picker.component.ts

```diff
@@ -337,9 +353,32 @@ export class SkyDateRangePickerComponent
       .subscribe(() => {
         this.#changeDetector.markForCheck();
       });
+
+    this.requiredState.requiredStateChange
```

Comment:
I had to add a `requiredStateChange` event to the directive. This is required because we need to update the outer form control once the required states change. With text-editor change detection was enough - but it is not here as we need validators to rerun and such in order to get all changes back to the outer control. We have to listen to when the required state changes, then update the inner controls to be required, then run the validators again for the outer control.